spec/features/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+feature 'user stuff' do
+
+  scenario 'I can sign up as a new user' do
+      user = build :user
+      expect { sign_up(user) }.to change(User, :count).by(1)
+      expect(page).to have_content('Welcome, foo@bar.com')
+      expect(User.first.email).to eq('foo@bar.com')
+    end
+
+    scenario 'I cannot sign up as a new user win non-matching passwords' do
+        user = build(:user,password:'wrong')
+        expect { sign_up(user) }.to change(User, :count).by(0)
+        expect(page).to have_content('Passwords do not match.')
+      end
```

Comment:
Lint/BlockAlignment: `end` at 14, 6 is not aligned with `scenario 'I cannot sign up as a new user win non-matching passwords' do` at 10, 4